src/commcare_cloud/ansible/roles/pg_repack/tasks/main.yml

```diff
@@ -62,23 +80,24 @@
     group: "postgres"
     mode: 0755
 
+- name: Copy pg_repack execution script
+  become: yes
+  template:
+    src: "pg_repack_cron.sh.j2"
+    dest: "/usr/local/sbin/pg_repack_cron.sh"
+    owner: "postgres"
+    group: "postgres"
+    mode: 0755
+  loop: "{{ pg_repack.get(inventory_hostname, []) }}"
+
 - name: Create pg_repack cron
   become: yes
   cron:
     name: "pg_repack {{ item.database }}"
-    job: >
-      "{{ pg_repack_script_path }} --pg-repack {{ postgres_install_dir }}/bin/pg_repack
-      --host {{ item.host|default('localhost') }}
-      --port {{ item.port|default(5432) }}
-      -d {{ item.database }}
-      {% if item.tables|default(None) %} --tables {{ item.tables|join(' ') }}{% endif %}
-      --table-size-limit {{ item.table_size|default(10) }}
-      --dead-tup-ratio {{ item.ratio|default(0.05) }}
-      {% if item.username|default(None) %} -U {{ item.username }}{% endif %}
-      {% if item.password|default(None) %} -W {{ item.password }}{% endif %}
-      {% if item.skip_superuser_check|default(pg_repack_skip_superuser_check) %} -k{% endif %}"
+    job: "/usr/local/sbin/pg_repack_cron.sh"
```

Comment:
same comment as above, you need to either pass in the parameters here or have a separate script file for each pg_repack job